src/main/java/de/tum/in/www1/artemis/repository/TextAssessmentEventRepository.java

```diff
@@ -23,4 +27,71 @@
     List<TextAssessmentEvent> findAllBySegmentType(TextBlockType type);
 
     List<TextAssessmentEvent> findAllByCourseId(Long courseId);
+
+    @Query("""
+            SELECT COUNT(DISTINCT textAssessmentEvent.userId)
+            FROM TextAssessmentEvent textAssessmentEvent
+            WHERE textAssessmentEvent.courseId = :#{#courseId} AND
+                textAssessmentEvent.textExerciseId = :#{#exerciseId}
+            """)
+    Integer getNumberOfTutorsInvolvedInAssessingByExerciseAndCourseId(Long courseId, Long exerciseId);
```

Comment:
I approved the newly added changes but decided to take a look at the code prior to the latest changes and noticed these SQL queries.
https://docs.artemis.ase.in.tum.de/dev/guidelines/server/
I think the @Param decorators are missing
![image](https://user-images.githubusercontent.com/65814168/133761129-c2c3772d-696c-47c4-b9c9-d8a29a65ff4e.png)
To have consistency with the documentation I suggest adding these decorators to all newly added methods.

Reply:
good catch, added changes